projects/addon-charts/utils/control-point.ts

```diff
@@ -4,18 +4,18 @@ import {tuiLineAngle} from './line-angle';
 import {tuiLineLength} from './line-length';
 
 export function tuiControlPoint(
-    current: TuiPoint,
+    current?: TuiPoint,
     previous?: TuiPoint,
     next?: TuiPoint,
     reverse = false,
     smoothing = 0.2,
 ): TuiPoint {
-    const p = previous || current;
-    const n = next || current;
+    const p = previous || current || [0, 0];
```

Comment:
Maybe we can add first line: `const fallback = current || [0, 0] as const` and use it down the line for readability?

Reply:
fixed